networking/hub-regionA.bicep

```diff
@@ -482,6 +482,26 @@ resource fwPolicy 'Microsoft.Network/firewallPolicies@2021-05-01' = {
                 '443'
               ]
             }
+            {
+              ruleType: 'NetworkRule'
+              name: 'pod-to-api-server_udp-1194'
+              description: 'This allows pods to communicate with the API server. Ensure your API server\'s allowed IP ranges support all of this firewall\'s public IPs.'
+              ipProtocols: [
+                'UDP'
+              ]
+              sourceAddresses: []
+              sourceIpGroups: [
+                ipgNodepoolSubnet.id
+              ]
+              destinationAddresses: [
+                'AzureCloud.${location}' // Ideally you'd list your AKS server endpoints in appliction rules, instead of this wide-ranged rule
+              ]
+              destinationIpGroups: []
+              destinationFqdns: []
+              destinationPorts: [
+                '1194'
+              ]
+            }
```

Comment:
Technically this is only needed for clusters that are not yet being created with konnectivty, which should be very few at this point.  We have hesitated to add it, because all the clusters that have konnectivity on them now would be getting added outbound firewall allowances that are not necessary. 

What would you say if you had this in here, but commented out with a comment that says "If you cluster does not deploy konnectivity, then this rule will also be needed."  That way for the handful of people that are still getting those clusters, they at least have a solution, and we're not putting a added firewall hole for a majority of people that are not needing the firewall hole?  I know it's not a great solution, either way. :(  Thoughts?

Reply:
Comment added, marking resolved.